htdocs/societe/card.php

```diff
@@ -1455,7 +1455,7 @@ function init_supplier_categ() {
 		// Type - Workforce/Staff
 		print '<tr><td>'.$form->editfieldkey('ThirdPartyType', 'typent_id', '', $object, 0).'</td><td class="maxwidthonsmartphone"'.($conf->browser->layout == 'phone' ? ' colspan="3"' : '').'>'."\n";
 		$sortparam = (empty($conf->global->SOCIETE_SORT_ON_TYPEENT) ? 'ASC' : $conf->global->SOCIETE_SORT_ON_TYPEENT); // NONE means we keep sort of original array, so we sort on position. ASC, means next function will sort on label.
-		print $form->selectarray("typent_id", $formcompany->typent_array(0), $object->typent_id, 0, 0, 0, '', 0, 0, 0, $sortparam, '', 1);
+		print $form->selectarray("typent_id", $formcompany->typent_array(0), $object->typent_id, 1, 0, 0, '', 0, 0, 0, $sortparam, '', 1);
```

Comment:
In the dictionary, there is currently a type TYPE_UNKNOWN that was dedicated when value is unknown so similar to nothing selected into the combo.
If we allow empty value as suggested (better), we should also remove the value unknown from dictionary. This may also have side effect on some join (inner join may need to be replaced into left join), so if we evolve to standardize code to remove the "unknown" value in this dictionary, we should do it into develop branch.

Reply:
I'm ok with change that in develop, but in 13.0 there is a regression, it's no more possible to have empty typeent... 
Or we remove ajax combobox for this field or we change the selecttypeent to return the empty  unknown value, or we merge this pr. I can plan some time to work on better implementation in develop. May be quicker and less dirty fix is to remove ajax combobox.

Reply:
I change by removing the select ajax
an PR in develop is comming with better fix